Compilation error on 0.15.0 with target wasm32-unknown-unknown

## Version

0.15.0

## Operating system & version

target: wasm32-unknown-unknown

## What you did

cargo check --target wasm32-unknown-unknown

## What you expected to happen

No compilation error

## What actually happened

```
PS C:\GitHub\leafwing-input-manager> **cargo check --target wasm32-unknown-unknown**
    Checking leafwing-input-manager v0.15.0 (C:\GitHub\leafwing-input-manager)
error[E0308]: arguments to this method are incorrect
   --> src\action_state\mod.rs:203:51
    |
203 |             .for_each(|action_datum| action_datum.tick(_current_instant, _previous_instant));
    |                                                   ^^^^ ----------------  ----------------- expected `std::time::Instant`, found `bevy::bevy_utils::Instant`
    |                                                        |
    |                                                        expected `std::time::Instant`, found `bevy::bevy_utils::Instant`
    |
    = note: `bevy::bevy_utils::Instant` and `std::time::Instant` have similar names, but are actually distinct types
note: `bevy::bevy_utils::Instant` is defined in crate `web_time`
   --> C:\Users\Fred\.cargo\registry\src\index.crates.io-6f17d22bba15001f\web-time-1.1.0\src\time\instant.rs:15:1
    |
15  | pub struct Instant(Duration);
    | ^^^^^^^^^^^^^^^^^^
note: `std::time::Instant` is defined in crate `std`
   --> C:\Users\Fred\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library/std/src/time.rs:156:1
    |
156 | pub struct Instant(time::Instant);
    | ^^^^^^^^^^^^^^^^^^
    = note: `bevy::bevy_utils::Instant` and `std::time::Instant` have similar names, but are actually distinct types
note: `bevy::bevy_utils::Instant` is defined in crate `web_time`
   --> C:\Users\Fred\.cargo\registry\src\index.crates.io-6f17d22bba15001f\web-time-1.1.0\src\time\instant.rs:15:1
    |
15  | pub struct Instant(Duration);
    | ^^^^^^^^^^^^^^^^^^
note: `std::time::Instant` is defined in crate `std`
   --> C:\Users\Fred\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library/std/src/time.rs:156:1
    |
156 | pub struct Instant(time::Instant);
    | ^^^^^^^^^^^^^^^^^^
note: method defined here
   --> src\action_state\action_data.rs:40:12
    |
40  |     pub fn tick(&mut self, _current_instant: Instant, _previous_instant: Instant) {
    |            ^^^^            -------------------------  --------------------------
```

## Additional information

PR incoming.

--- a/src/action_state/action_data.rs
+++ b/src/action_state/action_data.rs
@@ -1,8 +1,6 @@
 //! Contains types used to store the state of the actions held in an [`ActionState`](super::ActionState).
 
-use std::time::Instant;
-
-use bevy::{math::Vec2, reflect::Reflect};
+use bevy::{math::Vec2, reflect::Reflect, utils::Instant};
 use serde::{Deserialize, Serialize};
 
 #[cfg(feature = "timing")]

